src/app/facility/analysis/run-analysis/group-analysis/regression-model-selection/model-data-validation-filter.pipe.ts

```diff
@@ -0,0 +1,17 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { JStatRegressionModel } from 'src/app/models/analysis';
+
+@Pipe({
+  name: 'modelDataValidationFilter',
+  standalone: false
+})
+export class ModelDataValidationFilterPipe implements PipeTransform {
+
+   transform(models: Array<JStatRegressionModel>, showFailedDataValidation: boolean): Array<JStatRegressionModel> {
```

Comment:
Good use of this pipe. Could we combine this pipe with our other modelFilter pipe so there is only one filter pipe?